package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cl-react-graph",
-  "version": "3.0.0-alpha.40",
+  "version": "3.0.0-alpha.44",
```

Comment:
Is there a reason this has jumped from .40 to 44

Reply:
yeah I'd released a couple of test versions last night when no one was around - this MR contains all those changes